### PR TITLE
Bugfix #170235906 – Show fallback symbol for unknwon image value

### PR DESCRIPTION
**Pull request by alfonsomunozpomer**
If a column of type image has an unknown value, the table will not render and show an error message. This fix displays a span with a question mark in those cases.

### CODE REVIEW

File: __test__/TableContent.test.js [hunk 1]
```diff
@@ -82,7 +82,13 @@ describe(`TableContent`, () => {
 
   test(`should display the images in the table content if the header has image field`, () => {
     const wrapper = shallow(<TableContent {...props}/>)
-    expect(wrapper.find(`img`)).toHaveLength(props.tableHeader.length)
+    expect(wrapper.find(`img`)).toHaveLength(data.length - 1)
```
Comment (lingyun1010) [inline __test__/TableContent.test.js:85]: Thanks for the correction.

File: src/tableHeaderCells.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const TableSortHeaderCell = ({columnNumber, headerText, width, orderedColumnInde
     className={`header${columnNumber}`}
     flexBasis={width} flexShrink={100} flexGrow={100}
     onClick={() => onClick(columnNumber)}>
-    <div style={{display: `ruby`}}>
+    <div>
```
Comment (lingyun1010) [inline src/tableHeaderCells.js:10]: I added the `ruby` style to make sure the header text is displayed in one line. Are you sure you would like to delete it?

**Review by lingyun1010 — APPROVED** [review]
All good. Please check my comment on `ruby` style. If you are sure about that, please squash and merge it.

File: src/tableHeaderCells.js (hunk 2 above)
Comment (alfonsomunozpomer) [reply src/tableHeaderCells.js:10]: [Yes. :smile:](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-ruby-element)

**Pull request merged**